.github/workflows/linux.yml

```
@@ -13,6 +13,7 @@ jobs:
               "LCG_102/x86_64-centos9-gcc11-opt",    # g4 11
               "LCG_104/x86_64-el9-gcc13-opt",         # g4 11
               "LCG_105/x86_64-ubuntu2204-gcc11-opt",  # g4 11
+              "LCG_106/x86_64-el9-gcc13-opt",  # g4 11
```

```suggestion
              "LCG_104/x86_64-el9-gcc13-opt",  # g4 11
```
There we would actually check EDM4hep 0.10 instead of 0.10.5

---

But EIC is using 0.10.5 so that makes more sense? I understood Wouter's comment to build whatever latest patch version of 0.10.X if we were to build EDM4hep on the fly.

Otherwise 104 is already in the list above, and we could use that and just change the IF to enable EDM4hep.

---

> However, it should probably be 0.10, and not rely on a patch release.

I was coming from this angle and the fact that we have `find_package(EDM4HEP 0.10.0 REQUIRED)` in cmake. I think the main point is that the `find_package` and at least one CI check should be consistent for the actual minimal version.

---

But I am coming from the same angle? :confused: 

It is anyway good to add 106 based CI. We can additionally enable EDM4hep for 104.